index.html

```diff
@@ -20,15 +20,18 @@
          <a class="logo-link">My Logo</a> 
         </div>
         <nav class="menu hover">
-          <img src="img/menu.png" alt="Menu button">
+          <img class="menu-button" src="img/menu.png" alt="Menu button">
           <ul class="nav-links">
-            <li class="link-portfolio">
+            <li class="link-cancel d-none">
+              <img class="menu-button" src="img/Icon - Cancel.png" alt="Cancel icon" srcset="">
+            </li>
+            <li class="link-portfolio menu-button">
               <a href="#">Portfolio</a>
             </li>
-            <li class="link-about">
+            <li class="link-about menu-button">
               <a href="#">About</a>
             </li>
-            <li class="link-contact">
+            <li class="link-contact menu-button">
               <a href="#">Contact</a>
```

Comment:
- [ ] Nice work implementing the mobile menu, it looks almost identical to the Figma design :clap: :sunglasses: However the menu options are not link to any section. Kindly make sure when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed as requested in the project requirement.